rdm/xml_util.py

```diff
@@ -0,0 +1,63 @@
+from xml.etree import ElementTree
```

Comment:
I think all of the test record format python modules should go in a subdirectory, similar to the way that we put all of our renderer extensions in a subdirectory.

Reply:
Done.